### PR TITLE
Add April'23 Blog by Mind Benders & add authors

### PR DESCRIPTION
**Pull request by kunalagra**
This Month in AI is a monthly blog series maintained by Mind Benders Club that offers latest news in ML/AI/DS space.

This PR add the relevant blog posts for April'23, and add the authors involved in the making of the posts in author.yml.

### CODE REVIEW

File: blog/authors.yml [hunk 1]
```diff
@@ -44,4 +44,28 @@ jayesh:
...
+  name: Deexith Madas
+  title: Treasurer -  Mind Benders
+  url: https://github.com/MDeexith
+  image_url: https://github.com/MDeexith.png
+
+ananta:
+  name: Ananta Pandey
+  title: Jt. Publication Head - Mind Benders
+  url: https://github.com/Ananta564
+  image_url: https://github.com/Ananta564.png
```
Comment (demonkillerr) [inline blog/authors.yml:71]: I would highly recommend to add a pfp here, we discourage blanks

**Review by demonkillerr — CHANGES_REQUESTED** [review]

File: blog/authors.yml (hunk 1 above)
Comment (kunalagra) [reply blog/authors.yml:71]: Sure. Would Pass on the message to the team!

**Comment by kunalagra** [comment]
Moved authors to Index.md as per the new guidelines and made sure missing pfp have been added. Please review for any changes.

**Review by vish-rt — APPROVED** [review]
Ready to ship

**Pull request merged**